app/views/pages/new_features.html.erb

```diff
@@ -10,24 +10,33 @@
     <h2 class="govuk-heading-l">Recently released</h2>
     <p class="govuk-body">You can now:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>change locations assigned to a course (<a href="https://www.youtube.com/watch?v=W_zhfemKiYM" class="govuk-link">Watch a video<span class="govuk-visually-hidden">of the assigning locations feature</span></a>)</li>
       <li>edit course vacancies (<a href="https://www.youtube.com/watch?v=nxmtXGy1cCY" class="govuk-link">Watch a video<span class="govuk-visually-hidden">of the edit vacancies feature</span></a>)</li>
-      <li>add a new location</li>
-      <li>edit a location name and address</li>
-      <li>request a new course using a Google Form</li>
+      <li>edit a location name and address (<a href="https://www.youtube.com/watch?v=W_zhfemKiYM" class="govuk-link">Watch a video<span class="govuk-visually-hidden">of the assigning locations feature</span></a>)</li>
+      <li>add a new location (<a href="https://www.youtube.com/watch?v=W_zhfemKiYM" class="govuk-link">Watch a video<span class="govuk-visually-hidden">of the assigning locations feature</span></a>)</li>
+      <li>assign locations to a course (<a href="https://www.youtube.com/watch?v=W_zhfemKiYM" class="govuk-link">Watch a video<span class="govuk-visually-hidden">of the assigning locations feature</span></a>)</li>
+      <li>request a new course using a Google Form </li>
     </ul>
 
-    <h2 class="govuk-heading-l">May to June 2019</h2>
-    <p class="govuk-body">You’ll be able to:</p>
+    <h2 class="govuk-heading-l">July 2019</h2>
+    <p class="govuk-body">In July we’ll roll over your courses. Once we’ve done this, you’ll be able to:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>edit the UCAS Apply eligibility requirements for a course</li>
+      <li>view your courses for the next cycle (2020/21)</li>
+      <li>edit the contents of those courses</li>
+      <li>review their basic details</li>
+      <li>special educational needs and disabilities (SEND) information</li>
     </ul>
 
-    <h2 class="govuk-heading-l">July to September 2019</h2>
-    <p class="govuk-body">You’ll be able to:</p>
+    <h2 class="govuk-heading-l">August to September 2019</h2>
+    <p class="govuk-body">You’ll be able to edit all the basic details of your 2020/21 courses, including:</p>
```

Comment:
```suggestion
    <p class="govuk-body">You’ll be able to edit all the basic details of your courses, including:</p>
```
The edit features won't be specific to rollover.

Reply:
Actioned. 👍